src/unix/internal.h

```diff
@@ -106,6 +106,12 @@ int uv__pthread_sigmask(int how, const sigset_t* set, sigset_t* oset);
 # define UV__POLLRDHUP 0x2000
 #endif
 
+#ifdef POLLPRI
+# define UV__POLLPRI POLLPRI
+#else
+# define UV__POLLPRI 0x0002
```

Comment:
I'd define it to POLLIN to be sure it won't fail with EINVAL or something on systems without POLLPRI